src/tools/genpolicy/src/volume.rs

```diff
@@ -37,8 +37,7 @@ pub struct Volume {
     pub secret: Option<SecretVolumeSource>,
 
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub downwardAPI: Option<DownwardAPIVolumeSource>
-    // TODO: additional fields.
+    pub downwardAPI: Option<DownwardAPIVolumeSource>, // TODO: additional fields.
```

Comment:
Is this change required?

Reply:
Please use a separate PR for any such unrelated changes.

Reply:
Got it. Changes reverted.